src/main/java/hudson/model/RadiatorView.java

```diff
@@ -178,18 +183,10 @@ public ProjectViewEntry getContentsByPrefix()
 
 	private String getPrefix(String name) 
 	{
-		if (name.contains("_"))
+		if (name.contains(this.separatorChar))
 		{
-			return StringUtils.substringBefore(name, "_");
+			return StringUtils.substringBefore(name, this.separatorChar);
 		}		
```

Comment:
(Side note: I'm a bit reluctant to add yet another parameter to the constructor as explained in #18)

Anyway, there's a more important issue here IIUC: this change isn't backward-compatible. The code currently supports 3 separator characters, and your change only supports `_`.

IMO, your `separatorChar` should actually be an array `separatorChars` and you would then iterate on it (which is btw what the current code should have done instead of copy-paste... Anyway).